### ISSUE TITLE
Emply style element causes build error

### ISSUE THREAD

**Issue by nitro52** [kaisermann/svelte-preprocess#95]
**Describe the bug**
if there is an empty style element stubbed out i get a non descriptive build error
```
Module build failed (from ./node_modules/svelte-loader/index.js):
NonErrorEmittedError: (Emitted value instead of an instance of Error) [object Object]
```
**To Reproduce**
Add the line following line to a svelte project. Note this has been setup to use sass

```
<style type="text/scss"></style>
```
Notice that there are no spaces or line breaks within the style block. If you add a line break it fixes the build.

**Expected behavior**
build should not fail or give a better error

**Stacktraces**
<details>
  <summary>Stack trace</summary>
ERROR in ./src/App.svelte
Module build failed (from ./node_modules/svelte-loader/index.js):
NonErrorEmittedError: (Emitted value instead of an instance of Error) [object Object]
    at runLoaders (C:\Users\user\Documents\my-app\node_modules\webpack\lib\NormalModule.js:313:13)
    at C:\Users\user\Documents\my-app\node_modules\loader-runner\lib\LoaderRunner.js:367:11
    at C:\Users\user\Documents\my-app\node_modules\loader-runner\lib\LoaderRunner.js:233:18
    at context.callback (C:\Users\user\Documents\my-app\node_modules\loader-runner\lib\LoaderRunner.js:111:13)
    at preprocess.then.err (C:\Users\user\Documents\my-app\node_modules\svelte-loader\index.js:178:12)
 @ ./src/main.ts 1:0-31 2:16-19
 @ multi ./src/main.ts ./src/styles/default-theme/all.scss
</details>

**Information about your project:**
- Windows 10
- `svelte-preprocess` version (Please check you can reproduce the issue with the latest release!)
3.2.2

- Whether your project uses Webpack or Rollup
Wepack. based on my template i uploaded https://github.com/nitro52/svelte-typescript-sass-template. (might be slight tweaks to configs)

**Comment by nitro52** [comment]
I think this seems like an issue with the svelte-preprocessor but if it's not let me know.

**Comment by kaisermann** [comment]
Hello! Yeah, this was happening because `scss` throws an error if passed an empty string as content, who would know hahahaha 🤷‍♂

Anyway, fixed on `v3.2.5` 🎉
